### ISSUE TITLE
clean up destruct touched dead logic

### ISSUE THREAD

**Issue by zander-xyz** [category-labs/monad#371]
Related to thread: https://monadglobal.slack.com/archives/C03RKRT693K/p1695086366262489

> James Hunsaker
>   [1 hour ago](https://monadglobal.slack.com/archives/C03RKRT693K/p1695141802486459?thread_ts=1695086366.262489&cid=C03RKRT693K)
> if before spurious dragon there were empty accounts
> 
> 
> 
> 
> 
> James Hunsaker
>   [1 hour ago](https://monadglobal.slack.com/archives/C03RKRT693K/p1695141814273549?thread_ts=1695086366.262489&cid=C03RKRT693K)
> 2. after spurious dragon empty accounts are cleaned up if touched
> 
> 
> James Hunsaker
>   [1 hour ago](https://monadglobal.slack.com/archives/C03RKRT693K/p1695141834155309?thread_ts=1695086366.262489&cid=C03RKRT693K)
> 3. there appears to be a plan to initiate this for all empty accounts (in action, not in code)
> 
> 
> James Hunsaker
>   [1 hour ago](https://monadglobal.slack.com/archives/C03RKRT693K/p1695141844368299?thread_ts=1695086366.262489&cid=C03RKRT693K)
> 4. empty accounts are no longer created (??)
> 
> 
> James Hunsaker
>   [1 hour ago](https://monadglobal.slack.com/archives/C03RKRT693K/p1695141867243229?thread_ts=1695086366.262489&cid=C03RKRT693K)
> 5. the logic is no longer needed
> 
> 
> James Hunsaker
>   [1 hour ago](https://monadglobal.slack.com/archives/C03RKRT693K/p1695141890037029?thread_ts=1695086366.262489&cid=C03RKRT693K)
> for #3, we can just scan all the accounts and see if / how many empty accounts remain

TLDR; there seemed to be plan to send empty transactions to trigger the removal of empty accounts (per eip-161). see if that plan went into action, and if we can practically get rid of the `touched` logic based on our findings there.
